spec/services/sign_in/logger_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SignIn::Logger do
+  let(:logger) { SignIn::Logger.new }
+
+  describe '#info_log' do
+    let(:user_account) { create(:user_account) }
+    let(:user) { create(:user, uuid: user_account.id) }
```

Comment:
As an RSpec, tip, instead of doing `let(:user)` and then later the `before { allow(User).to receive(:find)... }`, use the `let!`, this will create the user before the test (instead of waiting for it to be used to create it), so it will naturally be available with a `User.find(uuid)` instead of having to mock it